model_analyzer/config/generate/base_model_config_generator.py

```diff
@@ -181,10 +197,63 @@ def make_model_config(param_combo, model, model_repository,
 
         return model_config
 
-    @staticmethod
-    def get_base_model_config_dict(model_repository, model_name):
-        config = ModelConfig.create_from_file(
-            f'{model_repository}/{model_name}')
+    @classmethod
+    def get_base_model_config_dict(cls, config, client, gpus, model_repository,
```

Comment:
I don't think this class should be the one who does all of this.  Can we push it down into ModelConfig class?

Reply:
Done